webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  mode: 'development',
+
+  entry: {
+    plugins: './presets/plugins.ts',
+  },
+
+  devtool: 'source-map',
+
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist')
+  },
+
+  plugins: [new webpack.ProgressPlugin()],
+
+  module: {
+    rules: [{
+      test: /.(ts|tsx)?$/,
+      loader: 'ts-loader',
```

Comment:
This is just a suggestion but do we want to look into `awesome-typescript-loader`, this way we can just point all components up to the main tsconfig flies rather than duplicating the same file in every package.